cmd/root.go

```diff
@@ -24,14 +29,29 @@ func Execute() {
 	}
 }
 
-var identifier string
-
 func init() {
 	// Here you will define your flags and configuration settings.
 	// Cobra supports persistent flags, which, if defined here,
 	// will be global for your application.
 
 	// Cobra also supports local flags, which will only run
 	// when this action is called directly.
-	rootCmd.Flags().StringVar(&identifier, "identifier", "devx", "Identifier for sharding manager instance")
+
+	smParams := model.ShardingManagerParams{}
+
+	rootCmd.PersistentFlags().StringVar(&smParams.KubeconfigPath, "kube_config", "", "Use a Kubernetes configuration file instead of in-cluster configuration")
+	rootCmd.Flags().StringVar(&smParams.ShardingManagerIdentity, "sharding-manager-identity", "devx", "Identity of the sharding manager instance")
```

Comment:
should we just call it shard?

Reply:
Whats the difference between `ShardingManagerIdentity`, `OperatorIdentityLabel`, `ShardIdentityLabel`?
Can you please add an example of where and how they will be used?